app/src/main/java/com/shub39/rush/domain/dataclasses/TTMLParser.kt

```
@@ -0,0 +1,392 @@
+                // Add agent info if present
+                val agentPrefix = if (!line.agent.isNullOrEmpty()) "{agent:${line.agent}}" else ""
+
+                appendLine(
+                    String.format(
+                        "[%02d:%02d.%02d]%s%s",
+                        minutes,
+                        seconds,
+                        centiseconds,
+                        agentPrefix,
+                        line.text,
+                    )
```

detekt (1.23.8)

[warning] String.format(
                        "[%02d:%02d.%02d]%s%s",
                        minutes,
                        seconds,
                        centiseconds,
                        agentPrefix,
                        line.text,
                    ) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)